manager/assets/modext/widgets/system/modx.grid.context.js

```diff
@@ -8,6 +8,17 @@
  */
 MODx.grid.Context = function(config) {
     config = config || {};
+    this.buttonColumnTpl = new Ext.XTemplate('<tpl for=".">'
```

Comment:
If we're going to apply this style of buttons to other grids (which for consistency would be a good idea), this property should be moved to the `MODx.grid.Grid` object to make it easily available. And added to the documentation. :)